css/tech.scss

```diff
@@ -0,0 +1,229 @@
+.tech{
+	.title {
+		text-align: center;
```

Comment:
The indents are really screwed up here until line 16, can you check that you're using 2 space indents, not 4 space tabs?